scripts/check_configs.ts

```diff
@@ -594,6 +603,26 @@ async function main() {
         }
       }
 
+      if(jsonData.market_banners) {
+        const marketBanners = jsonData.market_banners
+        const marketBannerIds = marketBanners.map((banner) => banner.market_id)
+
+        const hasInvalidMarketBannerIds = checkValidEntries(marketBannerIds, marketIds)
+        const hasDuplicateMarketBannerIds = checkDuplicateEntries(marketBannerIds)
+
+        if (hasInvalidMarketBannerIds.status && hasInvalidMarketBannerIds.entry) {
+          let listOfInvalidIds: string = hasInvalidMarketBannerIds.entry.join(", ");
+          console.error(`ERROR: ${network}.json has the following invalid market ids under the market_banners field: ${listOfInvalidIds}`)
+          outcomeMap[network] = false;
+        }
+
+        if (hasDuplicateMarketBannerIds.status && hasDuplicateMarketBannerIds.entry) {
+          let listOfDuplicates: string = hasDuplicateMarketBannerIds.entry.join(", ");
+          console.error(`ERROR: ${network}.json has duplicated market banners for the following market ids: ${listOfDuplicates}. Please make sure to input each market banner only once in ${network}`);
```

Comment:
Think you can change the 2nd sentence (i.e. `Please make sure to input...`) to the following:
```typescript
`Please make sure to add only 1 market banner for each market id in ${network}.json`.
```